fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -797,11 +805,14 @@ public void testGetWithExternalMessageMissingURLBinary() throws Exception {
                 .toString()
                 .replaceAll("<.*>", "< >");
 
-        final Response actual = testObj.createObject(null, null, null, null, Arrays.asList(badExternal), null);
+        testObj.createObject(null, null, null, null, Arrays.asList(badExternal), null);
```

Comment:
This is a side change.  I don't mind this so much, but you didn't do it for line 818 below.  (and I'm not sure if there are other spots).  I'm not clear on what's acceptable in a PR when it's outside the scope of the issue - it's a question I've had in general.  I'm a fan of cleaning code up, but I'm not sure if this should be reverted or if you should fix all the spots (and it might just be line 818, I'm not sure.).

Reply:
I was taking the approach of fixing warnings in methods that I updated for reasons related to the PR, but I was inconsistent and should have updated line 818 too in that case.